airbyte-integrations/connectors/source-google-ads/source_google_ads/source.py

```diff
@@ -69,6 +108,11 @@ def streams(self, config: Mapping[str, Any]) -> List[Stream]:
         incremental_stream_config = dict(
             api=google_api, conversion_window_days=config["conversion_window_days"], start_date=config["start_date"]
         )
+
+        custom_query_streams = [
```

Comment:
is it possible to validate custom queries in the `check_connection` method?

Reply:
used regex, since google query validator works on client (UI javascript) side